src/main/resources/reference.conf

```diff
@@ -38,4 +38,11 @@ videobridge {
       }
     }
   }
+  websockets {
+    enabled=false
+    # Must be set when enabled = true
+    # server-id="default-id"
```

Comment:
We could actually have a default value for this one

Reply:
You're right: my first thought was that these are either there or not (depending on if enabled is true), but when enabled _is_ true, a default makes sense here but not for, say, domain.  I forgot about that situation.

Reply:
What about the `tls` setting?